tests/qa/RS_VERSIONS-lite

```diff
@@ -0,0 +1,2 @@
+6.0.12-58
+100.0.0-2400
```

Comment:
This should be 100.0.0-2440 isn't it? @tomerhekredis ?

Reply:
On non-lite @tomerhekredis said we wanted 100.0.0-2400 on on RS_VERSIONS-<variant> the 6.0x thing.  It's worth noting we can now basically do anything here by adding variants and files. @tomerhekredis ?

Reply:
yes, it should be 2440 in all RS_VERSIONS files